### PR TITLE
Fixed & Updated Polish translation

### PR DESCRIPTION
**Pull request by LilSilv**
- Corrected Pawwit translation
- Fixed chat links
- Added apiTimeoutException, noRecentPlays, isSetId translation
- Renamed file to Polish.java

### CODE REVIEW

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+	@Override
+	public String unknownBeatmap() {
+		return "Przykro mi, nie rozpoznaje tej mapy. Możliwe że jest nowa, bardzo trudna, nierankingowa lub z innego trybu niż osu!standard";
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Ugh... Wygląda na to, że ludzki Tillerino uszkodził moje obwody."
+				+ "Gdyby wkrótce tego nie zauważył, mógłbyś go [https://github.com/Tillerino/Tillerinobot/wiki/Contact poinformować]? (odwołanie "
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java]: need a space here at the start of the string (before the G)

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.Response;
+
+/**
+ * Polish language implementation.
+ * Pawwit https://osu.ppy.sh/u/2070907 & LilSilv https://github.com/LilSilv https://osu.ppy.sh/users/8488688
+ */
+public class Default extends AbstractMutableLanguage {
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java]: oh and missed this, change `Default` to `Polish` here, it has to match the filename.

Also in `LanguageIdentifier.java` line 12 change `Polski.class` to `Polish.class`

Renaming the file gives headaches :eyes:

**Comment by omkelderman** [comment]
looks good to me, but travis is complaining, im confused...

```
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.5.0.1254:sonar (default-cli) on project tillerinobot: Unable to execute SonarQube: Fail to get bootstrap index from server: Failed to connect to localhost/127.0.0.1:9000: Connection refused (Connection refused) -> [Help 1]
```
@Tillerino halp?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java (hunk 2 above)
Comment (LilSilv) [reply tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java]: Done ^^

**Comment by Tillerino** [comment]
Right, so the whole sonar thing doesn't work with PRs. I'll have to put a condition in the Travis CI configuration. Let's do that another day...

**Pull request merged**